### DISCUSSION TITLE
Alert Distribution: Email / VOEvent

### DISCUSSION THREAD

**Discussion by fabianschussler** [chime-frb-open-data/community#36] [Q&A] [answered]
Hi!

I have a question and a comment concerning the distribution of real-time alerts via VoEvents and email:

Question: which email address will be used for the distribution of the alerts? The sign-up page asks for a 'contact' and a 'subscriber' address. Is it possible to have a third address that is used only for automatic treatment of alerts (and is not a contact nor a personal address of the subscriber)?

Comment: personally I would prefer to receive VoEvents, but our broker is not having a publicly accessible IP address. Why don't you distribute the alerts via the public VoEvent brokers (e.g. https://wiki.ivoa.net/twiki/bin/view/IVOA/IvoaVOEvent#VOEvent_Transport)?

Thanks a lot!
Fabian (on behalf of the [Astro-COLIBRI](https://astro-colibri.com/) team)

**Comment by andrewzwaniga** [comment, marked as answer]
Hi Fabian,

Thanks for your message. Regarding your question, the email version will be sent to the **subscriber** email. We also collect a contact email that would allow us to send occasional news about the Service (changes, updates, downtime, etc.).

The subscriber email is required to uniquely index entries in our subscriptions database. Some users are not interested in receiving the email version, so they may provide an email address that they do not use or check often for this part. In that case, the contact email is useful to us in order to still issue important news on occasion. 

Regarding your comment, we are asking users to subscribe directly to our broker with a public IP so that we can keep tally of the number of users and what our VOEvents are in use for. These are important metrics for us to secure resources that will allow us to continue producing the Service into the future. Publishing to a public broker that we do not control gives us less visibility into this.

We are happy to work with you to find a solution that would allow you to connect a broker to ours. If I understand correctly, you have a broker already that is subscribed to one or more of the existing public brokers. If that's the case, your broker should have a public IP, otherwise -- from my understanding -- the broadcaster would not be able to reach you. Perhaps we can help to sort this out?

Best regards,
Andrew

**Reply by fabianschussler** [reply]
Hi Andrew,

Thanks a lot for your reply!
I now signed up for the email based alerts...

Concerning the VoEvents: we are subscribed to several of the public VoEvent brokers (for redundancy) and receive the alerts distributed by them. These public services do not require the subscribers to have a static and public IP address (which we don't have) as there is no whitelisting/filtering. On our side we are using the comet broker to listen to the various alert streams...

Cheers,
Fabian
